keps/693-multikueue/README.md

```diff
@@ -223,7 +241,59 @@ If there is enough of global quota, the unknown admitted workloads would be re-a
 the management cluster. If not, some workloads will be preempted to meet the global quota.
 In case of duplicates, all but one of them will be removed.
 
-#### Follow ups ideas
+#### Garbage Collector
```

Comment:
Does this run periodically or react to some events?

Reply:
periodically, mentioned.